CMakeLists.txt

```diff
@@ -53,6 +53,23 @@ INCLUDE(CheckCXXCompilerFlag)
 INCLUDE(CheckCCompilerFlag)
 INCLUDE(CheckIncludeFile)
 
+SET (RTTR_RPATH "")
```

Comment:
Shouldn't this rather go into the `cmake/<platform>.cmake` files?
And what is `$ORIGIN`? Haven't heard of that.

Reply:
yea thats kind of interesting:
> $ORIGIN is a special variable that means ‘this executable’, and it means the actual executable  filename, as readlink would see it, so symlinks are followed.  In other words, $ORIGIN is special and resolves to wherever the binary is at runtime.

on work, we have for example `$ORIGIN/../lib` set, so a binary in `x/bin` will search for its libraries in `x/lib`